### PR TITLE
Add documentation for total collateral and return collateral.

### PR DESCRIPTION
**Pull request by Jimbo4350**
Related to: #3739

### CODE REVIEW

File: doc/reference/plutus/babbage-script-example.md [hunk 1]
```diff
@@ -12,6 +12,12 @@ An inline datum, is a datum that exists at a transaction output. We no longer ha
 
 In the case where we are not using a reference input to reference another transaction input (at a Plutus script address), we can specify a read only reference input that is simply exposed in the Plutus script context.
 
+## What are total and return collateral?
+
+The total collateral field lets users write transactions whose collateral is evident by just looking at the tx body instead of requiring information in the UTxO. The specification of total collateral is optional. It does not change how the collateral is computed but transactions whose collateral is different that this amount specified will be invalid.
+
+Return collateral allows us to specify an output with the remainder of our our collateral input(s) in the event we overcollateralize our transaction. This allows us to avoid overpaying the collateral.
```
Comment (james-iohk) [inline doc/reference/plutus/babbage-script-example.md:19]: Remove one "our"

**Review by james-iohk — APPROVED** [review]

**Review by kevinhammond — APPROVED** [review]
Looks good

**Review by dcoutts — APPROVED** [review]
LGTM

**Comment by Jimbo4350** [comment]
bors r -

**Pull request merged**